library/oneview_network_set.py

```diff
@@ -118,19 +118,21 @@ class NetworkSetModule(OneViewModule):
     MSG_ALREADY_PRESENT = 'Network Set is already present.'
     MSG_ALREADY_ABSENT = 'Network Set is already absent.'
     MSG_ETHERNET_NETWORK_NOT_FOUND = 'Ethernet Network not found: '
+    MSG_CONNECTION_TEMPLATE_RESET = 'Network Set connection template was reset to the default.'
```

Comment:
In which case do you return this message..?

Reply:
This also related to default bandwidth reset. Removed